source/mobile/mobile-overview.rst

```diff
@@ -1,37 +1,85 @@
 
-This option is recommended for:
+- What app will you deploy?
+- How will you deploy it?
 
-1. Organizations that want to customize the mobile apps
-2. Team Edition and Enterprise Edition servers that prefer to host their own push proxy server instead of using one of Mattermost's hosted versions
+Using Mattermost’s pre-built apps is the easiest and fastest approach since your users can download the apps from public app stores, including the `Apple App Store <https://www.apple.com/ca/app-store>`__ and the `Google Play Store <https://play.google.com/store>`__. Alternatively, you can use an EMM provider to enforce or restrict specific security policies. See `Using Mattermost’s Pre-Built Apps <https://docs.mattermost.com/mobile/use-prebuilt-mobile-apps.html>`__ to learn more about using Mattermost pre-built apps.
 
-Use an EMM provider with Managed App Configuration
----------------------------------------------------
+If you want to control the look, feel, or capabilities of the mobile apps, host your own push proxy server, or to maintain full control over app distribution, you can build your own mobile apps and manage your own app distribution. See `Building and Distributing Your Own Custom Mattermost Mobile Apps <https://docs.mattermost.com/mobile/build-custom-mobile-apps.html>`__ to learn more about working with custom apps.
 
-To meet other security and compliance policies, you can :doc:`deploy the Mattermost apps with any EMM provider <mobile-appconfig>` that supports `AppConfig <https://www.appconfig.org/members/>`__, such as MobileIron, Blackberry UEM, or Airwatch. You will still need to choose either the App Store/Google Play versions or apps compiled yourself to use in conjunction with the EMM provider.
+The following table summarizes the key differences between these two approaches:
 
-This option is recommended for organizations that typically use EMM solutions to deploy mobile apps to meet security and compliance policies.
++----------------------------------------------------------------+---------------------------------------------------------------------+
+| **Use Pre-Built Apps (Highly Recommended)**                    | **Build Custom Apps (Not Easy)**                                    |
++================================================================+=====================================================================+
+| **Recommended for:**                                           | **Recommended for:**                                                |
+|                                                                |                                                                     |
+| Self-supporting teams who need standard features               | Teams that need to customize the app, adhere to corporate           |
+| and have minimal corporate compliance rules.                   | compliance rules, or prefer to host their own push proxy server.    |
```

Comment:
Great question! @esethna - are there examples of corporate compliance rules available for cases where building a custom app is the better approach (that can't be achieved otherwise?)

Reply:
We can remove it. It's true that "compliance rules" are likely solved by appconfig variables which apply to both custom and pre-built apps